accounts/forms.py

```diff
@@ -179,6 +207,7 @@ def __init__(self, *args, **kwargs):
 
     class Meta:
         model = User
+        field_classes = {"username": ForceLowercaseEmailField}
```

Comment:
how come we need this here but not in ResearcherRegistrationForm?

Reply:
Meta is inheriting from `LowercaseUsernameUserCreationForm.Meta` in the other cases